### PR TITLE
README: better first glance idea

### LINKED ISSUE #347: hospitable README
could the buildah README.md have some copy paste example on running it, rather than so much build instructions?
it would seem much more useful to people searching for it and just giving it a 10second review
like, at first glance i would be 10x more likely to use/try a project if the README showed how it makes my life easier but having 5 pastes of compile instructions is a deterrent

### PR DESCRIPTION
**Pull request by vbatts**
Fixes #347

Make the project's first-glance easier to digest.

Signed-off-by: Vincent Batts <vbatts@hashbangbash.com>

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
FWIW, build error is a flake, couldn't talk to Fedora's registry.

File: README.md [hunk 1]
```diff
@@ -19,108 +19,40 @@ The Buildah package provides a command line tool which can be used to
...
-    skopeo-containers
-```
+Buildah uses `runc` to run commands when `buildah run` is used, or when `buildah build-using-dockerfile`
+encounters a `RUN` instruction, so you'll also need to build and install a compatible version of
+[runc](https://github.com/opencontainers/runc) for Buildah to call for those cases.
 
-Then to install Buildah on Fedora follow the steps in this example:
+## Example 
 
+From (`./examples/lighttpd.sh`)[examples/lighttpd.sh]:
```
Comment (TomSweeneyRedHat) [inline README.md]: This isn't showing as a link in the view page.  I think it needs to be:

From [examples/lighttpd.sh](`./examples/lighttpd.sh`):

Comment (TomSweeneyRedHat) [reply README.md]: and md just resolved that in my comment.  You need to have the stuff in the brackets before the stuff in the parens.

**Comment by rhatdan** [comment]
LGTM with link fixed.

**Comment by vbatts** [comment]
link fixed. PTAL

**Comment by vbatts** [comment]
would like this merged before kubecon gets fully underway today :-)

**Comment by TomSweeneyRedHat** [comment]
@nalind OK to merge?  Build issues are not related.

**Comment by nalind** [comment]
@TomSweeneyRedHat Yeah, it reads fine to me.  I'm going to have to bypass the merge bot.

**Pull request merged**